kalite/distributed/features/steps/superuser_create.py

```diff
@@ -0,0 +1,145 @@
+from behave import *
+from selenium.common.exceptions import NoSuchElementException
+
+from django.core.urlresolvers import reverse
+
+from kalite.testing.behave_helpers import *
+
+from kalite.testing.mixins.facility_mixins import CreateFacilityMixin, CreateStudentMixin
+
+
+from django.contrib.auth.models import User
+
+@given("there is superuser")
+def step_impl(context):
+    if User.objects.exists():
+        pass
+    else:
+        User.objects.create_superuser(username='superusername', password='superpassword', email='super@email.com')
+
+@then("there should be no modal displayed")
+def step_impl(context):
+    assert not find_id_with_wait(context, "superusercreate-container")
+
+@given("there is no superuser")
+def step_impl(context):
+    if User.objects.exists():
+        User.objects.all().delete()
+
+@then("I should see a modal")
+def step_impl(context):
+    assert find_id_with_wait(context, "superusercreate-container")
+
+@given("the username is empty")
+def step_impl(context):
+    fill_username(context, "")
+
+@when('I click the create button')
+def impl(context):
+    create_button = find_css_class_with_wait(context, "create-btn")
+    create_button.click()
+
+@then('the username border will turn red')
+def impl(context):
+    is_border_red(context, "super_username")
+
+@given("I enter a username longer than 30 letters")
+def step_impl(context):
+    fill_username(context, "123456789123456789123456789123456789")
+
+@when('I click the create button')
+def impl(context):
+    create_button = find_css_class_with_wait(context, "create-btn")
+    create_button.click()
+
+@then("the modal won't dismiss")
+def impl(context):
+    assert find_id_with_wait(context, "superusercreate-container")
```

Comment:
If the modal is dismissed, is it removed from the DOM? Or is it simple made invisible?